404.html

```diff
@@ -0,0 +1,123 @@
+                <div class="ui menu">
+                    <h1 class="item logo">
+                        <a href="./">OpenWISP</a>
+                    </h1>
+                    <div class="right menu">
+                        <div class="menu-open item">
+                            <i class="sidebar icon"></i>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="ui grid" id="main">
+            <div class="animation-area">
+                <ul class="box-area">
+                    <li></li>
+                    <li></li>
+                    <li></li>
+                    <li></li>
+                    <li></li>
+                    <li></li>
+                </ul>
+                <div class="ui center aligned text canvas-text">
+                    <h1 class=>Error 404!<br>
+                    </h1>
+                    <h3 class="error-msg">The page you are looking for doesn't exist.<br>
+                    </h3>
+                    <div class="ui hidden divider"></div>
+                    <p class="buttons center">
+                        <a href="index.html" class="ui big primary inverted button">
+                            <i class="icon home"></i>
+                            Home
+                        </a>
+                        <a href="support.html" class="ui big secondary inverted button">
+                            <i class="mail icon"></i>
+                            Support
+                        </a>
+                    </p>
+                </div>
+            </div>
+        </div>
+        <div class="ui black inverted segment padding-vertical" id="footer">
+            <div class="ui container stackable two column grid">
+                <div class="column logo white">
+                    <a href="./">OpenWISP</a>
+                </div>
+                <div class="column social">
+                    <a href="https://twitter.com/openwisp" class="ui twitter button">
+                        <i class="twitter icon"></i>
+                        Twitter
+                    </a>
+                    <a href="https://facebook.com/openwisp" class="ui facebook button">
+                        <i class="facebook icon"></i>
+                        Facebook
+                    </a>
+                    <a href="https://www.linkedin.com/company/openwisp" class="ui linkedin button">
+                        <i class="linkedin icon"></i>
+                        Linked In
+                    </a>
+                    <a href="https://github.com/openwisp" class="ui black button">
+                        <i class="github icon"></i> Github
+                    </a>
+                </div>
+                <p>&copy; 2008-
+                    <script>document.write(new Date().getFullYear())</script> OpenWISP and individual contributors.
+                </p>
+            </div>
+        </div>
+    </div>
+    </div>
```

Comment:
You can remove this tag (`/div`) since tags must be paired, i.e., no start tag `<div>`.